src/main/fc/fc_tasks.c

```diff
@@ -183,7 +185,7 @@ static void taskUpdateBaro(timeUs_t currentTimeUs)
 }
 #endif
 
-#if defined(USE_BARO) || defined(USE_SONAR)
+#if defined(USE_ALT_HOLD)
```

Comment:
This is not only used for altitude hold, but also for altitude display in OSD / telemetry / blackbox. I think if we disable this then we can just as well disable BARO and SONAR and not build all related drivers.

Reply:
You're right. I don't want to disable display of altitude, just altitude hold code. I'll update.